frontend/test/__support__/e2e/config.js

```diff
@@ -71,6 +71,8 @@ const defaultConfig = {
     config.env.SOURCE_VERSION = sourceVersion;
     config.env.TARGET_VERSION = targetVersion;
 
+    require("@deploysentinel/cypress-debugger/plugin")(on, config);
```

Comment:
Need to put this after `before:browser:launch` custom listener to avoid conflict